documentation/graphing-overview/index.html.md

```diff
@@ -0,0 +1,279 @@
+
+	<html>
+		<head>
+			<link rel="stylesheet" href="https://forio.com/tools/contour/contour.min.css">
+			<script src="yourPath/d3.min.js"></script>
+			<script src="yourPath/lodash.js"></script>
+			<script src="https://forio.com/tools/contour/contour.min.js"></script>
+		</head>
+		
+		<body>
+		</body>
+		
+	</html>
+
+Including `contour.js` from the top-level `forio.com/tools/contour/` directory guarantees you'll always be using the latest version. If you want to work with a particular version of Contour, you can include the version number in the path:
+	
+	<script src="https://forio.com/tools/contour/1.0.1/contour.min.js"></script>
+
+See more about <a href="http://forio.com/contour/documentation.html#quickstart" target="_blank">including Contour</a>.
+
+**Step 2: Create a new Contour chart on the page.**
+
+In JavaScript on the page of your project where the visualization appears, create a new Contour instance:
+
+	var myChart = new Contour({
+			el: '.myChart'
+		})
+		.cartesian()
+		.line()
+		.tooltip();
+
+This code:
+
+* Calls the Contour constructor, passing it a set of options. The `el` option is required; it is the selector of the container in which the Contour instance will eventually be rendered.
+* Sets the frame for this set of visualizations (`.cartesian()` is the only currently available frame; it is required for all Contour visualizations except pie charts, so remove this line if you are making a pie chart).
+* Adds two specific visualizations to this Contour instance: `line()` and `tooltip()`. The visualizations to add to your Contour chart are all optional, and which ones to include depends on what your final visualization should look like.
+
+Note that unlike many common (static) uses of Contour, this code is *not* rendering the Contour instance just yet.
+
+See more about <a href="http://forio.com/contour/documentation.html#key_concepts" target="_blank">creating a new Contour chart</a>.
+
+**Step 3: Use the Variables Channel to subscribe to the model variable you want to graph.**
+
+In Flow.js, you can use the [Variables Channel](../generated/channels/variables-channel/) to receive notifications when a model variable is updated. Because we want to update our visualization as a model variable changes, we subscribe to the variable. 
+
+	Flow.channel.variables.subscribe('myVariable', function() { });
+
+
+**Step 4: In the callback function for your subscription, update and render the Contour chart.**
+
+The Flow.js [Variables Channel `subscribe()`](../generated/channels/variables-channel/) method's second argument is a callback function, called whenever the model variable is updated. We want to update and re-render our visualization each time the model variable changes:
+
+	Flow.channel.variables.subscribe('myVariable',
+		function(data) {
+			myChartData.push(data.myVariable);
+			myChart.setData(myChartData);
```

Comment:
I don't think we can if we also want the simplest example to use `channel.variables.subscribe` AND user input instead of an operation. 

In the multiple series example, the user clicks a button to advance the model; we just happening to be listening to two vars that change when the model is advanced. In other words, to get around `push`, we need the model to have an operation that appends to the array and records, and then that means you're subscribing to the operation not the variable. 

Do you mean that the simplest case should be an operation, rather than user input?